docker/rucio_client/scripts/user_to_site_mapping.py

```diff
@@ -110,7 +110,14 @@ def set_rucio_limits(cric_user):
         print("Add account for %s %s" % (account, email))
 
         try:
-            client.get_account(account)
+            rucio_account = client.get_account(account)
+            if rucio_account['email'] != email:
+                # Update the account:
+                if dry_run:
+                    print('DRY-RUN: CRIC email is different from rucio email. Rucio email should be updated')
+                else:
+                    print('CRIC email is different from rucio email. Updating')
+                    client.update_account(account, 'email', email)
         except AccountNotFound:
             client.add_account(account, cric_user.account_type, email)
```

Comment:
Looks like the necessary functionality is added, thanks Keane. I only have comment: Since you added the `dry_run` option to the `set_rucio_limits` function, I would apply it to all "write/update" calls in this function which seem to be `add_account` , `add_scope` and `set_local_account_limit`. Otherwise, it would be inconsistent.

Reply:
How about the function `add_identities_to_rucio` in line 130?

Reply:
I checked what it actually does [1] and looks like it indeed does updates in rucio, thus I'd say let's also add dry-run option there. @ericvaandering what do you think?

[1] https://github.com/dmwm/CMSRucio/blob/681cf868d0a0f89cca3922cb7bbfba010dbb23d4/docker/rucio_client/scripts/cric_user.py#L71